PipelinesTasks/PRMetrics/buildAndReleaseTask/Updaters/CodeMetrics.ts

```diff
@@ -0,0 +1,429 @@
+/* eslint-disable spaced-comment */
```

Comment:
File names should begin with a lowercase letter as per convention. You might need to delete the files, push and then re-add them to correct the case though